packages/react-input/src/components/Input/__snapshots__/Input.test.tsx.snap

```diff
@@ -2,12 +2,16 @@
 
 exports[`Input renders a default state 1`] = `
 <div>
-  <div
+  <span
```

Comment:
Is it expected to have `span`s there? Why not `div`s?

Reply:
It's intentional (there's a comment about this in `renderInput`). If the input is inline within a `p` there will be nesting errors if I use `div`.

In a previous version of the implementation I used `div` by default and `span` if `inline` was true, but someone suggested just using `span` always since it (probably?) doesn't make much practical difference (all the elements either have `display` explicitly set, or are within a flex layout). I'm open to suggestions here though, and could go back to that if there's some reason it would be better.